client/src/pages/Home/Footer/Footer.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+import "./Footer.css";
+import { FacebookOption, Twitter, Instagram } from "grommet-icons";
+
+export function Footer() {
+	const [isShown, setIsShown] = useState(false);
+	return (
+		<div id="footer">
+			{/* Section11 */}
+			<div className="home_section11_parent">
+				<div className="home_section11_child1">
+					<img
+						className="footer-logo"
+						src="https://i.postimg.cc/VkdMYP4S/Logo.jpg"
+						alt="logo"
+					/>
+				</div>
+				<div className="home_section11_child2">
+					<div>
+						<p>Do want to be part of us?</p>
+					</div>
+					<div>
+							<a
+								href="https://example.com/"
+								target="_blank"
+								rel="noreferrer"
+								className="footer-button"
+								onMouseEnter={() => setIsShown(true)}
+								onMouseLeave={() => setIsShown(false)}
+							>
+								Get started
+							</a>
+						{isShown && <div className="footer_hidden-show"> Get started</div>}
+					</div>
+				</div>
+			</div>
+
+			{/* section12 */}
+			<div className="home_section12_parent">
+				<div className="home_section12_child1">
+					<ul>
+						<li>List</li>
+						<li>List</li>
+						<li>List</li>
+					</ul>
+					<ul>
+						<li>List</li>
+						<li>List</li>
+						<li>List</li>
+					</ul>
+					<ul>
+						<li>List</li>
+						<li>List</li>
+						<li>List</li>
+					</ul>
+				</div>
+				<div className="home_section12_child2">
+					<p>Our Email address</p>
+					<span>info@sensational-life.com</span>
```

Comment:
can you make the email anchor with mailto: href

Reply:
I'm sorry that I missed deleting the following file.
Footer.js Footer.css I need to delete the folder.

the email icon is in Section12, so I will add the icon in another component, thank you.